mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowProtobufMapper.java

```diff
@@ -118,6 +121,68 @@ String makeRestoreRun(String runId) {
     return print(builder);
   }
 
+  String makeGetLatestVersion(String modelName) {
+      return makeGetLatestVersion(modelName, null);
+  }
+
+  String makeGetLatestVersion(String modelName, String stage) {
+    RegisteredModel model = RegisteredModel.newBuilder()
+            .setName(modelName)
+            .build();
+
+    GetLatestVersions.Builder builder = GetLatestVersions.newBuilder();
+    builder.setRegisteredModel(model);
+
+    if (stage != null) {
+      builder.addStages(stage);
```

Comment:
surprised this takes in a single stage

Reply:
I've updated the interface to take in a set of stages now.